backend/server.py

```
@@ -4876,6 +4876,29 @@ class ProviderUpdate(BaseModel):
     priority: int = Field(default=None, ge=-100, le=1000)
 
 
+
+# BUG-19 fix: Provider policy endpoints (paid-provider kill switch)
+@app.get("/api/providers/policy")
+async def get_provider_policy_route(user: dict = Depends(get_current_user)):
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)